eng/Versions.props

```diff
@@ -13,4 +13,8 @@
     <MicrosoftDotNetXUnitExtensionsVersion>8.0.0-beta.25105.2</MicrosoftDotNetXUnitExtensionsVersion>
     <MicrosoftDotNetBuildTasksTargetFrameworkVersion>8.0.0-beta.25105.2</MicrosoftDotNetBuildTasksTargetFrameworkVersion>
   </PropertyGroup>
+  <!-- SDK dependencies -->
+  <PropertyGroup>
+      <MicrosoftDotNetApiCompatTaskVersion>10.0.100-preview.3.25124.13</MicrosoftDotNetApiCompatTaskVersion>
```

Comment:
No need to introduce this property here. Versions.props properties are only needed for Maestro updated dependencies.

Reply:
Fixed